Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorEffects/AbstractScriptedSegmentEditorAutoCompleteEffect.py

```diff
@@ -66,6 +66,10 @@ def __del__(self):
         self.delayedAutoUpdateTimer.stop()
         self.observeSegmentation(False)
 
+    def cleanup(self):
+        self.delayedAutoUpdateTimer.disconnect("timeout()", self.onPreview)
+
+
```

Comment:
```suggestion
```

Reply:
Updated the last commit to include this change